### PR TITLE
checkfiles() update 1

### PR DESCRIPTION
**Pull request by looseMole**
Broken links are now deleted, if user acts on them.

### CODE REVIEW

File: src/domain/domain.py [hunk 1]
```diff
@@ -22,7 +22,8 @@ def upload_file(self, filepath, encryption_method, server):
 		if not self.fh.upload_file(encrypted_file_path, server, file_hash, encryption_key, encryption_method):
 			return False
 
-		return os.remove(encrypted_file_path)
+		os.remove(encrypted_file_path)
```
Comment (TidosDK) [inline src/domain/domain.py]: A try catch should probably be implemented here

File: src/data/file_handler.py [hunk 2]
```diff
@@ -214,22 +215,44 @@ def check_files(self):
...
-            for j in range(1, len(self.uploaded_files[key]), 3):
+            for j in range(1, len(self.uploaded_files[key]), 3):  # For download link related to file.
+                enc_method = self.uploaded_files[key][j + 2]
+
                 # Deconstruct download URL
                 download_url = self.uploaded_files[key][j]
                 string_list = download_url.split('/')
 
                 # Build info URL:
+                print(string_list)
```
Comment (TidosDK) [inline src/data/file_handler.py]: Is this relevant for the user?

File: src/presentation/CLI.py [hunk 3]
```diff
@@ -40,9 +41,88 @@ def main(self):
...
+
+                                string_list = good_url.split('/')
+                                good_down_name = string_list[2]
+                                break
+
+                        print("\n\"" + uploaded_files[key][0] + "\"" + " link: " + broken_url + " is broken.")
+
+                        if amount_of_valid_links > 0:
+                            print(
+                                "Would you like to download \"" + file_name + "\" from " + good_down_name +
```
Comment (TidosDK) [inline src/presentation/CLI.py:77]: good_down_name variable might not be initialized.

**Review by TidosDK — COMMENTED** [review]
Please see my review questions/notes.
Other than that, it looks good.

File: src/domain/domain.py (hunk 1 above)
Comment (looseMole) [reply src/domain/domain.py]: Will do

File: src/data/file_handler.py (hunk 2 above)
Comment (looseMole) [reply src/data/file_handler.py]: No, forgot that one

File: src/presentation/CLI.py (hunk 3 above)
Comment (looseMole) [reply src/presentation/CLI.py:77]: Not really, but I will adress it for the sake of our Linters

**Comment by looseMole** [comment]
Comments addrssed. How does it look, @TidosDK?

**Review by TidosDK — COMMENTED** [review]
I looks all fine now👍

**Pull request merged**